crates/biome_analyze/src/rule.rs

```diff
@@ -440,6 +448,8 @@ impl RuleDomain {
             ],
             Self::Solid => &[],
             Self::Next => &[],
+            Self::Vue => &[],
+            Self::Vue3 => &[],
```

Comment:
Vue has globals, which are here (for now): 

https://github.com/biomejs/biome/blob/90cc52bd5bffa56e708155994da7ff73a577ef09/crates/biome_service/src/file_handlers/javascript.rs#L328-L340

We should add them here, and remove them from `javascript.rs`